markdown/3.0/en/api/ff-asn.api.md

```diff
@@ -50,14 +43,14 @@ ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-| **selected**&nbsp;(String)&nbsp;**Options**:&nbsp;"true",&nbsp;"false" (default: empty) | The alignment of the After Search Navigation. |
+| **selected**&nbsp;(String)&nbsp;**Options**:&nbsp;"true",&nbsp;"false" (default: "false") | The alignment of the After Search Navigation. |
 | **element**&nbsp;(Object)&nbsp;(default: empty) | The filter data. |
 | **group**&nbsp;(Object)&nbsp;(default: empty) | The filter group data. |
 
 ### Methods
 | Name | Description |
 | ---- | ----------- |
-| **clone** | Return a deep copy of the contained elements including all properties, behaviors, private fields, states and HTML templates. |
+| **clone(group, element)** | Returns a copy of the group element with the specified group and element to provide the new context. |
```

Comment:
I wonder about the reason for making `clone` part of the public api - what is the use case, where a customer would use it?

I think in the long term the `clone` will be obsolete due to lit migration and generating HTML declarative  from data instead of this imperative `clone`. So I think _3.0_ would be a nice timing to drop that from public interface

Reply:
I have that topic on my list. At the moment I don't have enough information to decide if it can be removed safely. Didn't push it any further due to lack of time until release.